inference-engine/samples/speech_sample/main.cpp

```diff
@@ -857,7 +859,11 @@ int main(int argc, char *argv[]) {
 
                 uint32_t numFramesReference(0), numFrameElementsReference(0), numBytesPerElementReference(0),
                         numBytesReferenceScoreThisUtterance(0);
-                const uint32_t numScoresPerFrame = ptrOutputBlob.size() / batchSize;
+                const uint32_t numScoresPerFrame = outputs.empty() ?
+                        InferenceEngine::details::product(cOutputInfo.rbegin()->second->getDims()) / batchSize :
+                        InferenceEngine::details::product(cOutputInfo[outputs[next_output]]->getDims()) / batchSize;
+
```

Comment:
Instead of includes you can make it
auto dims = outputs.empty() ? cOutputInfo.rbegin()->second->getDims() : cOutputInfo[outputs[next_output]]->getDims();
const uint32_t numScoresPerFrame = std::accumulate(std::begin(dims), std::end(dims), (size_t)1, std::multiplies<size_t>());